vmdb/app/models/ems_refresh.rb

```diff
@@ -55,16 +55,13 @@ def self.refresh(target, id = nil)
 
     # Split the targets into refresher groups
     groups = targets.group_by do |t|
-      # Determine the group
       ems = t.respond_to?(:ext_management_system) ? t.ext_management_system : t
-      if t.respond_to?(:emstype)
-        ems.kind_of?(EmsVmware) ? :vc : ems.emstype.to_sym
-      end
+      ems.kind_of?(EmsVmware) ? "vc" : ems.emstype.to_s
     end
 
     # Do the refreshes
-    [:vc, :ec2, :rhevm, :scvmm, :kvm, :openstack, :openstack_infra].each do |g|
```

Comment:
I honestly can't remember why I did it this way, because your code makes so much more sense.  I think I was trying to force a particular ordering, but oh well.